server/test/mocks/tMock.js

```diff
@@ -1,15 +0,0 @@
-// Third-party dependencies
```

Comment:
I felt this file was redundant and could be written into individual test case files where need be.

Reply:
OK.